contributing/REPOSITORY-GUIDELINES.md

```diff
@@ -0,0 +1,69 @@
+# Knative Repository Guidelines
+
+This document outlines a structure for creating and associating code repositories
+with the Knative project. It also describes how and when repositories are removed.
+
+## Core Repositories
+
+Core repositories are considered core components of Knative. They are utilities, tools,
+applications, or libraries that form or support the foundation of the project.
+
+Core repositories are all those located under the
+[github.com/knative organization](https://github.com/knative).
+
+### Core Repository Requirements
+
+* Repository must live under github.com/knative/project-name
+* Must adopt the Knative Code of Conduct
+* All code projects must use the Apache License version 2.0.
+* Documentation repositories must use Creative Commons License version 4.0.
+* All OWNERS must be members of the Knative community.

```

Comment:
What does it mean to be a member of the community? What specific action/verification is there?

Reply:
I'll clarify -- they must be members of knative-dev.